serveradmin/common/templatetags/serversearch.py

```diff
@@ -22,7 +22,11 @@ def serversearch_js(search_id):
     return {
         'attributes_json': json.dumps(attributes),
         'filters_json': json.dumps(
-            [f.__name__ for f in filter_classes if f != ExactMatch]
+            [
+                f.__name__
+                for f in filter_classes
+                if f != ExactMatch and not f.__doc__.startswith('Deprecated')
```

Comment:
Not like the idea to use such dynamic thing as docstring to filter deprecated stuff. Wonder if we could provide a decorator for that or just directly introduce a property to mark it as deprecated.

Reply:
It is just temporary.  The next merge request I promise will get rid of this as well.  I am putting a `TODO` in here too.